src/extension/mixer.ts

```diff
@@ -0,0 +1,162 @@
+import OSC from 'osc-js';
+import { get } from './util/nodecg';
+import { TaggedLogger } from './util/tagged-logger';
+import { Channel } from 'src/types/custom';
+
+const nodecg = get();
+
+const config = nodecg.bundleConfig.mixer;
+const log = new TaggedLogger('mixer');
+const channelIdToName = {
+  "1": "H1",
+  "2": "H2",
+  "3": "H3",
+  "4": "H4",
+  "9": "Host1",
+  "10": "Host2",
+  "16": "Host3",
+  "5": "Gra1",
+  "7": "Gra2",
+  "11": "Donacje",
+};
+const channelNameToId = Object.assign({}, ...Object.entries(channelIdToName).map(([k, v]) => ({ [v]: k })))
+const mixerSignalLevels = nodecg.Replicant<{ [key in Channel]: number }>('mixerSignalLevels', { defaultValue: {
+  "": -Infinity,
+  "H1": -Infinity,
+  "H2": -Infinity,
+  "H3": -Infinity,
+  "H4": -Infinity,
+  "Host1": -Infinity,
+  "Host2": -Infinity,
+  "Host3": -Infinity,
+  "Gra1": -Infinity,
+  "Gra2": -Infinity,
+  "Donacje": -Infinity,
+}});
+
+// TODO add some panel to control them in case we need to tweak them live
+const mixerThresholdLevels = nodecg.Replicant<{ [key in Channel]: number }>('mixerThresholdLevels', { defaultValue: {
```

Comment:
```suggestion
// @ts-ignore
const mixerThresholdLevels = nodecg.Replicant<{ [key in Channel]: number }>('mixerThresholdLevels', { defaultValue: {
```

this can be done instead of a dummy read to avoid the build error, couldn't find a better way

Reply:
I removed the assignment completely, just left the call to the Replicant without and it doesn't complain